stix_shifter_modules/okta/stix_transmission/connector.py

```diff
@@ -172,7 +166,7 @@ async def ping_connection(self):
                 return_obj = self.exception_response(response_code, response_dict.get('errorSummary', ''))
 
         except Exception as ex:
-            if "timeout_error" in str(ex) or "client_connector_error" in str(ex):
+            if "timeout_error" in str(ex):
                 response_dict['code'] = 300
```

Comment:
300 generally is not a request timeout error. you should set 408 for this. And set equivalent error message in the stix_shifter_modules/okta/stix_transmission/error_mapper.py

Reply:
Updated the response status code as suggested.